services/people.ts

```diff
@@ -9,6 +9,7 @@ export interface IPerson {
   linkedin?: string;
   themeColor?: string;
   roles?: string[];
+  customImage?: boolean;
```

Comment:
`customImage` need to be the name of the image, and not using github prop for the picture
that way we not blading variables and what they need to do

Reply:
Uri wrote that this should be  a true or false variable.